### PR TITLE
AWS S3 Actions - Adding new documentation on supported integrations, and mappings

### PR DESCRIPTION
**Pull request by jpang007**
### Proposed changes
Added in extra documentation on mapping functionality as well as the supported Segment Feature Integrations 

### Merge timing
ASAP once approved

### Related issues (optional)
N/A

### CODE REVIEW

File: src/connections/destinations/catalog/actions-s3/index.md [hunk 1]
```diff
@@ -25,6 +25,12 @@ The AWS S3 (Actions) destination enhances this capability by introducing configu
 
 * **Enhanced Delivery Controls**. The destination provides advanced options for batch size controls and file naming conventions. These controls can help optimize efficiency and simplify data retrieval workflows.
 
+## Supported Integrations
+The AWS S3 (Actions) Destination supports the following Segment features as supported native Destination integration points: 
+* rETL
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-s3/index.md:30]:
```suggestion
* [Reverse ETL](/docs/connections/reverse-etl/)
```

File: src/connections/destinations/catalog/actions-s3/index.md [hunk 2]
```diff
@@ -25,6 +25,12 @@ The AWS S3 (Actions) destination enhances this capability by introducing configu
...
+* Classic and Linked Audiences
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-s3/index.md:31]:
```suggestion
* [Classic and Linked Audiences](/docs/engage/audiences/) 
```

File: src/connections/destinations/catalog/actions-s3/index.md [hunk 3]
```diff
@@ -25,6 +25,12 @@ The AWS S3 (Actions) destination enhances this capability by introducing configu
 
 * **Enhanced Delivery Controls**. The destination provides advanced options for batch size controls and file naming conventions. These controls can help optimize efficiency and simplify data retrieval workflows.
 
+## Supported Integrations
+The AWS S3 (Actions) Destination supports the following Segment features as supported native Destination integration points: 
+* rETL
+* Classic and Linked Audiences 
+* Connections
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-s3/index.md:32]:
```suggestion
* [Connections](/docs/connections/)
```

File: src/connections/destinations/catalog/actions-s3/index.md [hunk 4]
```diff
@@ -36,7 +42,7 @@ Ensure you have the following in place before configuring the AWS S3 (Actions) d
 - IAM Access IDs: Prepare your AWS IAM ARN ID and IAM External ID. These will be needed to authenticate and authorize Segment with your S3 bucket.
 
 
-### Step 1: Create an IAM role in the AWS console
+## Step 1: Create an IAM role in the AWS console
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-s3/index.md:45]: I would prefer to keep all of the setup steps one heading level under the Getting started header to keep a consistent pattern across other integrations!

```suggestion
### Step 1: Create an IAM role in the AWS console
```

File: src/connections/destinations/catalog/actions-s3/index.md [hunk 5]
```diff
@@ -81,8 +87,9 @@ To set up the IAM role to properly authorize Segment with the AWS S3 (Actions) d
    ]
  }
 ```
+
 ## Step 2: Add the AWS S3 (Actions) Destination in Segment
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-s3/index.md:91]:
```suggestion
### Step 2: Add the AWS S3 (Actions) Destination in Segment
```

File: src/connections/destinations/catalog/actions-s3/index.md [hunk 6]
```diff
@@ -93,8 +100,15 @@ To finish the configuration, enable the AWS S3 (Actions) Destination in your wor
    * Enter the ARN of the IAM role you created above. The ARN should follow the format `arn:aws:iam::ACCOUNT_ID:role/ROLE_NAME.`
    * Enter the IAM External ID, which is a value set in the Trust Relationship under your AWS IAM Role.
 5. Enable the destination.
-6. Verify the Segment data is stored in the S3 bucket by navigating to the `<your_S3_bucket>/` in the AWS console. 
 
 {% include components/actions-fields.html settings="true"%}
 
+## Step 3: Configure the AWS S3 (Actions) Destination mappings
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-s3/index.md:106]:
```suggestion
### Step 3: Configure the AWS S3 (Actions) Destination mappings
```

File: src/connections/destinations/catalog/actions-s3/index.md [hunk 7]
```diff
@@ -93,8 +100,15 @@ To finish the configuration, enable the AWS S3 (Actions) Destination in your wor
...
+To finish the configuration, finalize the mappings of the new AWS S3 (Actions) Destination: 
 
+1. Add a new **Sync to S3** Action into the destination. 
+2. Define the Event Trigger. If multiple types are accepted in the Event Trigger, the generated files will automatically be split by type in S3 (i.e Track events file and Identifer events file).
+3. Configure the Column Mappings that are needed, if any of the default columns are **not** needed, leave the value blank. You can also choose to add new mapping fields to set up any customized columns as needed. 
+4. Configure the Other Settings as needed.
+5. Enable the Mapping and start sending data! 
+6. Verify the Segment data is stored in the S3 bucket by navigating to the `<your_S3_bucket>/` in the AWS console.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-s3/index.md:114]:
```suggestion
6. Verify that Segment is sending data to your S3 bucket by navigating to `<your_S3_bucket>/` in the AWS console. 
```

File: src/connections/destinations/catalog/actions-s3/index.md [hunk 8]
```diff
@@ -93,8 +100,15 @@ To finish the configuration, enable the AWS S3 (Actions) Destination in your wor
...
 {% include components/actions-fields.html settings="true"%}
 
+## Step 3: Configure the AWS S3 (Actions) Destination mappings
+To finish the configuration, finalize the mappings of the new AWS S3 (Actions) Destination: 
 
+1. Add a new **Sync to S3** Action into the destination. 
+2. Define the Event Trigger. If multiple types are accepted in the Event Trigger, the generated files will automatically be split by type in S3 (i.e Track events file and Identifer events file).
+3. Configure the Column Mappings that are needed, if any of the default columns are **not** needed, leave the value blank. You can also choose to add new mapping fields to set up any customized columns as needed. 
+4. Configure the Other Settings as needed.
+5. Enable the Mapping and start sending data!
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-s3/index.md:113]:
```suggestion
5. Enable the Mapping.
```

File: src/connections/destinations/catalog/actions-s3/index.md [hunk 9]
```diff
@@ -93,8 +100,15 @@ To finish the configuration, enable the AWS S3 (Actions) Destination in your wor
...
 
 {% include components/actions-fields.html settings="true"%}
 
+## Step 3: Configure the AWS S3 (Actions) Destination mappings
+To finish the configuration, finalize the mappings of the new AWS S3 (Actions) Destination: 
 
+1. Add a new **Sync to S3** Action into the destination. 
+2. Define the Event Trigger. If multiple types are accepted in the Event Trigger, the generated files will automatically be split by type in S3 (i.e Track events file and Identifer events file).
+3. Configure the Column Mappings that are needed, if any of the default columns are **not** needed, leave the value blank. You can also choose to add new mapping fields to set up any customized columns as needed. 
+4. Configure the Other Settings as needed.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-s3/index.md:112]:
```suggestion
4. Configure any additional settings as required.
```

File: src/connections/destinations/catalog/actions-s3/index.md [hunk 10]
```diff
@@ -93,8 +100,15 @@ To finish the configuration, enable the AWS S3 (Actions) Destination in your wor
...
-6. Verify the Segment data is stored in the S3 bucket by navigating to the `<your_S3_bucket>/` in the AWS console. 
 
 {% include components/actions-fields.html settings="true"%}
 
+## Step 3: Configure the AWS S3 (Actions) Destination mappings
+To finish the configuration, finalize the mappings of the new AWS S3 (Actions) Destination: 
 
+1. Add a new **Sync to S3** Action into the destination. 
+2. Define the Event Trigger. If multiple types are accepted in the Event Trigger, the generated files will automatically be split by type in S3 (i.e Track events file and Identifer events file).
+3. Configure the Column Mappings that are needed, if any of the default columns are **not** needed, leave the value blank. You can also choose to add new mapping fields to set up any customized columns as needed.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-s3/index.md:111]:
```suggestion
3. Configure the Column Mappings. If you don't need any of the default columns, leave the value blank. You can also choose to add new mapping fields to set up customized columns as needed. 
```

File: src/connections/destinations/catalog/actions-s3/index.md [hunk 11]
```diff
@@ -93,8 +100,15 @@ To finish the configuration, enable the AWS S3 (Actions) Destination in your wor
...
 5. Enable the destination.
-6. Verify the Segment data is stored in the S3 bucket by navigating to the `<your_S3_bucket>/` in the AWS console. 
 
 {% include components/actions-fields.html settings="true"%}
 
+## Step 3: Configure the AWS S3 (Actions) Destination mappings
+To finish the configuration, finalize the mappings of the new AWS S3 (Actions) Destination: 
 
+1. Add a new **Sync to S3** Action into the destination. 
+2. Define the Event Trigger. If multiple types are accepted in the Event Trigger, the generated files will automatically be split by type in S3 (i.e Track events file and Identifer events file).
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-s3/index.md:110]:
```suggestion
2. Define the Event Trigger. If multiple types are accepted in the Event Trigger, the generated files will automatically be split by type in S3 (for example, you might have a Track events file and an Identify events file).
```

File: src/connections/destinations/catalog/actions-s3/index.md [hunk 12]
```diff
@@ -93,8 +100,15 @@ To finish the configuration, enable the AWS S3 (Actions) Destination in your wor
    * Enter the ARN of the IAM role you created above. The ARN should follow the format `arn:aws:iam::ACCOUNT_ID:role/ROLE_NAME.`
    * Enter the IAM External ID, which is a value set in the Trust Relationship under your AWS IAM Role.
 5. Enable the destination.
-6. Verify the Segment data is stored in the S3 bucket by navigating to the `<your_S3_bucket>/` in the AWS console. 
 
 {% include components/actions-fields.html settings="true"%}
 
+## Step 3: Configure the AWS S3 (Actions) Destination mappings
+To finish the configuration, finalize the mappings of the new AWS S3 (Actions) Destination:
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-s3/index.md:107]:
```suggestion
To finish the configuration, add mappings to your new AWS S3 (Actions) Destination: 
```

**Review by tcgilbert — APPROVED** [review]

File: src/connections/destinations/catalog/actions-s3/index.md (hunk 5 above)
Comment (tcgilbert) [reply src/connections/destinations/catalog/actions-s3/index.md:91]: @forstisabella one of the reasons @jpang007 submitted this was to get all the steps in the side-nav: 
Screenshot 2024-12-16 at 3 47 15 PM

I think we should either show all in side-nav or none. That said, I will defer to @forstisabella on what we should use as the standard

Comment (forstisabella) [reply src/connections/destinations/catalog/actions-s3/index.md:91]: That makes sense! I think we shouldn't have the steps in the side nav to stay consistent with other destinations' docs!

File: src/connections/destinations/catalog/actions-s3/index.md (hunk 2 above)
Comment (jpang007) [reply src/connections/destinations/catalog/actions-s3/index.md]: done

File: src/connections/destinations/catalog/actions-s3/index.md (hunk 3 above)
Comment (jpang007) [reply src/connections/destinations/catalog/actions-s3/index.md]: done

File: src/connections/destinations/catalog/actions-s3/index.md (hunk 4 above)
Comment (jpang007) [reply src/connections/destinations/catalog/actions-s3/index.md]: done

File: src/connections/destinations/catalog/actions-s3/index.md (hunk 5 above)
Comment (jpang007) [reply src/connections/destinations/catalog/actions-s3/index.md]: done

File: src/connections/destinations/catalog/actions-s3/index.md (hunk 8 above)
Comment (jpang007) [reply src/connections/destinations/catalog/actions-s3/index.md]: done

File: src/connections/destinations/catalog/actions-s3/index.md (hunk 9 above)
Comment (jpang007) [reply src/connections/destinations/catalog/actions-s3/index.md]: done

File: src/connections/destinations/catalog/actions-s3/index.md (hunk 10 above)
Comment (jpang007) [reply src/connections/destinations/catalog/actions-s3/index.md]: done

File: src/connections/destinations/catalog/actions-s3/index.md (hunk 11 above)
Comment (jpang007) [reply src/connections/destinations/catalog/actions-s3/index.md]: done

File: src/connections/destinations/catalog/actions-s3/index.md (hunk 12 above)
Comment (jpang007) [reply src/connections/destinations/catalog/actions-s3/index.md]: done

**Review by forstisabella — APPROVED** [review]

**Review by tcgilbert — APPROVED** [review]

**Pull request merged**